stockflux-securities-master/src/components/SecuritiesTable.js

```diff
@@ -1,93 +1,149 @@
-import React, { useEffect, useState } from "react";
+import React, { useEffect, useState, useCallback } from "react";
 import Components from "stockflux-components";
-import "./SecuritiesTable.css";
 import { Link } from "react-router-dom";
-import { getSecuritiesData } from "../services/SecuritiesService";
-import AddSecurityButton from "./AddSecurityButton";
+import "./SecuritiesTable.css";
+import {
+  getSecuritiesData,
+  deleteSecurity
+} from "../services/SecuritiesService";
+import ValidationError from "../services/ValidationError";
 import Alert from "./Alert";
+import { TableState } from "../enums";
+import Button from "./Button";
 
-const SecuritiesTable = () => {
+const tableBody = (onClickDelete, securitiesData, state) => {
```

Comment:
I think maybe you should have this inside `SecuritiesTable` as it very tightly related to Securitites table. It would alos save you passing in these arguments `tableBody(onClickDelete, securitiesData, state)` and you could just do `{tableBody}` rather than `{tableBody(onClickDelete, securitiesData, state)}`.

Another reason for this would be readability. Normally the first thing you would expect to see is the definition of the object that filename refers to, in this case "SecuritiesTable", rather than `tableBody`.

Reply:
I see, good point. I have now done this